src/bin/sccache-dist/main.rs

```diff
@@ -254,6 +254,7 @@ fn check_server_token(server_token: &str, auth_token: &str) -> Option<ServerId>
 #[derive(Serialize, Deserialize)]
 #[serde(deny_unknown_fields)]
 struct ServerJwt {
+    exp: usize,
```

Comment:
Shouldn't this be a fixed size type? `u64` or `u128`?

Reply:
Right, changed to `u64` as in `JobJwt`.